tests/tests_files_elasticsearch_certs_incomplete.yml

```diff
@@ -47,7 +47,6 @@
         include_role:
           name: linux-system-roles.logging
 
-      - name: unreachable task
```

Comment:
Why are you deleting this? There has to be a list of tasks under the block and you are removing the `- ` list item marker.

Reply:
> Why are you deleting this? There has to be a list of tasks under the block and you are removing the `- ` list item marker.

Yeah, obviously this patch is broken :( Sorry for pushing it too early.

Reply:
> > Why are you deleting this? There has to be a list of tasks under the block and you are removing the `- ` list item marker.
> 
> Yeah, obviously this patch is broken :( Sorry for pushing it too early.

@pcahyna, I've updated the patch. All I had to do was adding 2 column indentation... :p